android-vjestina-homework-movie-app-data-mock/app/src/main/res/values/strings.xml

```diff
@@ -1,3 +1,4 @@
 <resources>
     <string name="app_name">MovieApp</string>
+    <string name="sf">Sci-Fi</string>
```

Comment:
Where do we need this resource?

Reply:
No need to extract then
Preview is used for development purposes only so you can hardcode this value in the preview function